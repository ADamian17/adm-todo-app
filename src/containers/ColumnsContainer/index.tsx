import Columns from "../../components/Columns";
import Hero from '../../components/Hero'
import ThemeProvider from "../../components/ThemeProvider";

const ColumnsContainer = () => {
  return (
    <>
      <ThemeProvider />
      <Hero />

      <Columns />
    </>
  )
};

export default ColumnsContainer;