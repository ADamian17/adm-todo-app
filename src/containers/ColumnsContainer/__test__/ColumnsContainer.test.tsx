import { render } from '@testing-library/react';

import ColumnsContainer from '..';

Object.defineProperty(window, "matchMedia", {
  writable: true,
  value: jest.fn().mockImplementation((query) => ({
    matches: false,
    media: query,
    onchange: null,
    addListener: jest.fn(), // deprecated
    removeListener: jest.fn(), // deprecated
    addEventListener: jest.fn(),
    removeEventListener: jest.fn(),
    dispatchEvent: jest.fn(),
  })),
});

describe('App', () => {
  it('renders headline', () => {
    const tree = render(<ColumnsContainer />);

    expect(tree).toMatchSnapshot()
  });
});