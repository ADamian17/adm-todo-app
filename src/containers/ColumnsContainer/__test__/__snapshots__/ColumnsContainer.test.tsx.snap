// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders headline 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="hero"
        style="background-image: url(images/bg-desktop-light.jpg);"
      >
        <nav
          class="nav"
        >
          <svg
            class="headline"
          >
            <use
              href="/icons/main-icons.svg#todo"
            />
          </svg>
          <svg
            class="icon"
          >
            <use
              href="/icons/main-icons.svg#moon"
            />
          </svg>
        </nav>
        <form
          class="wrapper"
          data-testid="add-todo-form"
        >
          <div
            class="inputWrapper"
          >
            <input
              class="textField false"
              data-testid="add-todo-form-input"
              placeholder="e.g oil change"
              type="text"
              value=""
            />
          </div>
          <input
            class="submitBtn"
            data-testid="add-todo-form-btn"
            type="submit"
            value="add todo"
          />
        </form>
      </header>
      <section
        class="wrapper"
      >
        <section
          class="column"
        >
          <p
            class="headline"
          >
            todo
          </p>
          <ul
            class="todos false"
            data-rbd-droppable-context-id="0"
            data-rbd-droppable-id="todo"
          />
        </section>
        <section
          class="column"
        >
          <p
            class="headline"
          >
            doing
          </p>
          <ul
            class="todos false"
            data-rbd-droppable-context-id="0"
            data-rbd-droppable-id="doing"
          />
        </section>
        <section
          class="column"
        >
          <p
            class="headline"
          >
            done
          </p>
          <ul
            class="todos false"
            data-rbd-droppable-context-id="0"
            data-rbd-droppable-id="done"
          />
        </section>
      </section>
    </div>
    <div
      aria-atomic="true"
      aria-live="assertive"
      id="rbd-announcement-0"
      style="position: absolute; width: 1px; height: 1px; margin: -1px; border: 0px; padding: 0px; overflow: hidden; clip-path: inset(100%);"
    />
    <div
      id="rbd-hidden-text-0-hidden-text-0"
      style="display: none;"
    >
      
  Press space bar to start a drag.
  When dragging you can use the arrow keys to move the item around and escape to cancel.
  Some screen readers may require you to be in focus mode or to use your pass through key

    </div>
  </body>,
  "container": <div>
    <header
      class="hero"
      style="background-image: url(images/bg-desktop-light.jpg);"
    >
      <nav
        class="nav"
      >
        <svg
          class="headline"
        >
          <use
            href="/icons/main-icons.svg#todo"
          />
        </svg>
        <svg
          class="icon"
        >
          <use
            href="/icons/main-icons.svg#moon"
          />
        </svg>
      </nav>
      <form
        class="wrapper"
        data-testid="add-todo-form"
      >
        <div
          class="inputWrapper"
        >
          <input
            class="textField false"
            data-testid="add-todo-form-input"
            placeholder="e.g oil change"
            type="text"
            value=""
          />
        </div>
        <input
          class="submitBtn"
          data-testid="add-todo-form-btn"
          type="submit"
          value="add todo"
        />
      </form>
    </header>
    <section
      class="wrapper"
    >
      <section
        class="column"
      >
        <p
          class="headline"
        >
          todo
        </p>
        <ul
          class="todos false"
          data-rbd-droppable-context-id="0"
          data-rbd-droppable-id="todo"
        />
      </section>
      <section
        class="column"
      >
        <p
          class="headline"
        >
          doing
        </p>
        <ul
          class="todos false"
          data-rbd-droppable-context-id="0"
          data-rbd-droppable-id="doing"
        />
      </section>
      <section
        class="column"
      >
        <p
          class="headline"
        >
          done
        </p>
        <ul
          class="todos false"
          data-rbd-droppable-context-id="0"
          data-rbd-droppable-id="done"
        />
      </section>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
